Comments/ClientApp/src/components/Document/Document.scss

```diff
@@ -45,6 +45,13 @@
   }
 }
 
+.caution {
+  background-color: $colour-caution-our-version;
+  border: 2px solid black ;
+  display: inline-block;
+  padding: 10px;
```

Comment:
10px of padding. how about using the design system's spacing variables instead?

Reply:
I hadn't seen that spacing stuff, thanks!